src/mempool.rs

```diff
@@ -378,14 +366,18 @@ impl Mempool {
         Ok(())
     }
 
-    /// Emits a DataProposals event containing data proposals between the two cuts provided.
+    /// Emits a SignedBlock event matching.
     /// If data is not available locally, fails and do nothing
-    fn handle_fetch_data_proposals(&mut self, from: Option<&Cut>, to: &Cut) -> Result<()> {
+    fn handle_ccp_interval(&mut self, interval: &CcpInterval) -> Result<()> {
```

Comment:
This would be clearer if it returned data proposals, and we sent the message in the calling function.
Then it would be renamed `try_get_full_data_for_signed_block`